arquillian-recorder-reporter/arquillian-recorder-reporter-api/src/main/resources/arquillian_reporter_templates/en/arquillian_reporter_template.xsl

```diff
@@ -172,49 +172,75 @@
-      #hoverpopup { visibility:hidden; position:absolute; top:0; left:0; color: white; background-color: black;}
-      .testClass.collapsed>.testMethod,.containers.collapsed>.container,.testMethod.collapsed>div,.configuration.collapsed>div { display: none; }
-      .collapsable:hover { cursor: pointer; }
-      .collapsable::before { content: "[-] "; }
-      .collapsable:hover::before { color: red; }
-      .collapsed .collapsable::before { content: "[+] "; }
+      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+      <title>Arquillian test run report</title>
+      <style type="text/css">
+          body { margin: 1em; padding: 0; }
+          h1 { font-size: 300%; margin: 0; padding: 0; display: inline-block; }
+          h1 > img { float:left; margin: 0 5px 0 0; }
+          h2 { font-size: 275%; margin: 0; padding: 0; }
+          h3 { font-size: 250%; margin: 0; padding: 0; }
+          h4 { font-size: 200%; margin: 0; padding: 0; }
+          h5 { font-size: 150%; margin: 0; padding: 0; display: inline-block; }
+          h5 > img { float:left; margin: 0 5px 0 0; }
+          h6 { font-size: 125%; margin: 0; padding: 0; }
+          .clear { clear: both; }
+          .extensions { margin: 0 0 0 2em; }
+          .extension { margin: 0 0 0 2em; }
+          .suite,.configuration { margin: 0 0 0 2em; }
+          .containers { margin: 0 0 0 2em; }
+          .container { margin: 0 0 0 2em; }
+          .deployments { margin: 0 0 0 2em; }
+          .deployment { margin: 0 0 0 2em; }
+          .testClasses { margin: 0 0 0 2em; }
+          .testClass { margin: 0 0 0 2em; }
+          .testMethod { margin: 0 0 0 2em; }
+          .testMethod > table { margin: 0 0 0 2em; }
+          .properties { margin: 0 0 0 2em; }
+          .properties > table { margin: 0 0 0 2em; }
+          .video { margin: 0 0 0 2em; }
+          .video > video { margin: 0 0 0 2em; }
+          .screenshotParent { margin: 0 0 0 2em; }
+          .screenshotLeft { display: inline-block; float: left; }
+          .screenshotLeft:nth-child(1n) { margin: 0 0 0 2em; }
+          .passed { color: #336633; }
+          .failed { color: #FF0000; }
+          .skipped { color: #FF9900; }
+          .whitespaces { margin: 0 0 0 2em; white-space: pre; }
+          .exception_stacktrace { overflow: hidden; height: 16%; }
+          .exception_stacktrace::first-line { font-weight:bold; font-size: 110%; }
+           #hoverpopup { visibility:hidden; position:absolute; top:0; left:0; color: white; background-color: black;}
+          .testClass.collapsed>.testMethod,.containers.collapsed>.container,.testMethod.collapsed>div,.configuration.collapsed>div { display: none; }
+          .collapsable:hover { cursor: pointer; }
+          .collapsable::before { content: "[-] "; }
+          .collapsable:hover::before { color: red; }
+          .collapsed .collapsable::before { content: "[+] "; }
+          #hor-minimalist-a
```

Comment:
I would suggest using css classes rather than `id`s. Ids should be unique and not really used for carrying visual information.

Maybe we can also think about some better name for the class which would reveal it's look&feel.